themes/default/content/templates/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Pulumi Templates
+meta_desc: This is a totally lovely meta description that still needs to be written.
+---
+
+<!-- Here is a page that describes what templates are, why they're awesome, which ones we have, and where to go from here. -->
+
+Pulumi templates are the fastest way to deploy a running application to AWS, Azure, and Google Cloud. After deploying, you can easily modify the program by updating the code in TypeScript, Python, Go, Java, .NET/C#, or YAML.
```

Comment:
@gthuang I might suggest we tweak this a bit: Templates won't always deploy "running applications"; often, they'll deploy only infrastructure, and a website doesn't really qualify as an application, either.

Reply:
Agree fully. I’ve seen this pop up in numerous places. These are the fastest way to _create an infrastructure as code project for your scenario_.

Reply:
roll with this. I made two edits. 

Pulumi templates are the fastest way to deploy infrastructure to AWS, Azure, and Google Cloud. After deploying, you can easily modify the infrastructure by updating the code in TypeScript, Python, Go, Java, .NET/C#, or YAML.